components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -576,7 +636,8 @@ def getConfigValue(key):
         rectW = int((w+xOffset) * factor)
         rectH = int((h+yOffset) * factor)
         draw.ellipse((rectX, rectY, rectW, rectH), outline=lineColour)
-        draw.ellipse((rectX-1, rectY-1, rectW+1, rectH+1), outline=lineColour) # hack to get line width of 2
+        draw.ellipse((rectX-1, rectY-1, rectW+1, rectH+1), outline=lineColour)
+        # hack to get line width of 2
```

Comment:
Sorry, in my last comment on this I should have explicitly marked the other cases of the inverted comments.